cmd/server/server.go

```diff
@@ -40,7 +40,8 @@ import (
 	"github.com/woodpecker-ci/woodpecker/remote"
 	"github.com/woodpecker-ci/woodpecker/router"
 	"github.com/woodpecker-ci/woodpecker/router/middleware"
-	droneserver "github.com/woodpecker-ci/woodpecker/server"
+	"github.com/woodpecker-ci/woodpecker/server"
+	woodpeckerGrpcServer "github.com/woodpecker-ci/woodpecker/server/grpc"
```

Comment:
Could this be just `grpc`?

Reply:
No. We're already importing `"google.golang.org/grpc"` so we need to disambiguate somehow.

Reply:
`grpcServer` was also not available since we're using that var. We could call it `grpcImplementation` or `realGrpcServer`.

Reply:
this soulds like we should at some point move `"google.golang.org/grpc"` into our own grpc moduel ??

for now please leave it - this would be more than just code move and split I guess

Reply:
Or we could play with the purpose of it. This grpc server is for communication between the woodpecker-server and woodpecker-agent. Maybe `agentGrpc`?

Reply:
We can just leave it for now. At some point we need to refactor the var names of this file containing drone anyways.